src/vs/workbench/parts/preferences/common/keybindingsEditorModel.ts

```diff
@@ -90,19 +90,72 @@ export class KeybindingsEditorModel extends EditorModel {
 
 	public fetch(searchValue: string, sortByPrecedence: boolean = false): IKeybindingItemEntry[] {
 		searchValue = searchValue.trim();
-		const quoteAtFirstChar = searchValue.charAt(0) === '"';
-		const quoteAtLastChar = searchValue.charAt(searchValue.length - 1) === '"';
+
+		if (this.isSourceFilterApplied(searchValue)) {
+			searchValue = this.getSourceFilterValue(searchValue);
+			searchValue = this.parseSearchValue(searchValue);
+			return this.fetchKeybindingItemsBySource(sortByPrecedence ? this._keybindingItemsSortedByPrecedence : this._keybindingItems, searchValue, this.quoteAtFirst(searchValue) && this.quoteAtLast(searchValue));
```

Comment:
Not sure why you need compute quotes here?

Reply:
For the same reasons as above.